packages/vitaminjs-jest/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "vitaminjs-jest",
+  "version": "1.0.0",
+  "description": "Plugin for vitamin that enable testing with jest",
+  "repository": "https://github.com/Evaneos/vitaminjs/tree/master/packages/vitaminjs-plugin-jest",
+  "keywords": [
+    "vitaminjs",
+    "jest"
+  ],
+  "collaborators": [
+    "Johan Girod <johangirod@gmail.com>",
+    "Lucas Personnaz <lucas@evaneos.com"
```

Comment:
You're email is missing `>` at the end. Aren't you the only author of this package?